### ISSUE TITLE
release milestone 6.1.1

### ISSUE THREAD

**Issue by nicoa** [jazzband/pip-tools#1422]
Hey,

some words upfront: if this is somehow considered annoying, feel free to close this issue without comment!
TL;DR: is making the release on my own appreciated or not?

As I'm not aware of the process I wanted to kindly ask how and when the releases are determined to be "ready" to be done and by whom. In this case, there is already another release (6.2.0) drafted and I'd be happy to see the contents of 6.1.1. on pip.

While I know that your time (addressing maintainers) is rare and you may consider such inquiries as disturbing as we all do it for free and you do the releases whenever there will be the time, I'd be happy to support here by filing a PR for that specific one. 

Unfortunately I didn't found documentation on this and it seems that always the same people file those release PRs, thus asking if that's appreciated or rather not.

Edit: if it is simply going to be merged with 6.2, I'll wait for that to pass. Just curious about what is going to happen where :)

**Comment by atugushev** [comment]
`6.2.0` release is on the way #1423

**Comment by nicoa** [comment]
Great, so it will be included in that one. Then I didn't say anything, thanks! :)
